pkg/traversal/traversal.go

```diff
@@ -84,13 +85,19 @@ func (w *FileWalker) processDir(inputDir input.Directory, outputDirName string)
 			}
 			// If the file should be omitted then stop processing.
 			if omit {
+				if klog.V(2).Enabled() {
+					klog.Infof("omitting '%s'", e.Path())
```

Comment:
Can't this just be `klog.V(2).Infof()`?

Reply:
ah you're right, I was looking at the docs the wrong way. I thought the latter wouldn't skip the locking. Fixing